cli/src/java/org/apache/hadoop/hive/cli/CliDriver.java

```diff
@@ -250,7 +250,8 @@ CommandProcessorResponse processLocalCmd(String cmd, CommandProcessor proc, CliS
         }
 
         // Set HDFS CallerContext to queryId and reset back to sessionId after the query is done
-        ShimLoader.getHadoopShims().setHadoopQueryContext(qp.getQueryState().getQueryId());
+        ShimLoader.getHadoopShims()
+            .setHadoopQueryContext(qp.getQueryState().getQueryId() + " User: " + ss.getUserName());
```

Comment:
I wonder if we should avoid embedding spaces in the format. Prior usage of caller context that I've seen uses an underscore-delimited format. The Hadoop compatibility guidelines state that the HDFS audit log format should be considered public and stable:

https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/Compatibility.html#Audit_Log_Output

Embedding spaces could break existing scripts that perform positional parsing using utilities like `cut` and `awk`.

Reply:
Thanx for the pointers, no specific reason for using space, if it can break parsers, better to avoid!!